pkgs/by-name/ex/extism-js/package.nix

```diff
@@ -0,0 +1,74 @@
+# Building this from source requires cross-compiling Rust to wasm32-wasip1.
+# An attempt to do so was made in https://github.com/NixOS/nixpkgs/pull/463349.
+# FIXME revisit once https://github.com/NixOS/nixpkgs/pull/463720 is merged
+
+{
+  autoPatchelfHook,
+  fetchurl,
+  lib,
+  libgcc,
+  stdenvNoCC,
+  versionCheckHook,
+}:
+
+let
+  version = "1.5.1";
+  tag = "v${version}";
+  system = lib.replaceStrings [ "darwin" ] [ "macos" ] stdenvNoCC.hostPlatform.system;
+  hashes = {
+    aarch64-darwin = "sha256-BHld8Dwwd6fexc05oHOIawa5PtGZAI61wQGWE8T+iMs=";
+    aarch64-linux = "sha256-KM3/y31OdLmEAcc48TSLmXei2GD6FhOHYlD7W/ErP+I=";
+    x86_64-darwin = "sha256-cE0JJK92pTJyGnHEZ7wA6+dpMvVOTMzFM2Mkwfy5kbQ=";
+    x86_64-linux = "sha256-9qSjqPFmUaUnpGQ/ldIpyFSgTWbUYozcckpYxpm5PJU=";
```

Comment:
rip freebsd, lol. Hopefully by the time I care about making this work again we've merged 463349.